Common/Securities/Cash.cs

```diff
@@ -138,7 +138,7 @@ public Security EnsureCurrencyDataFeed(SecurityManager securities, SubscriptionM
         {
             if (Symbol == CashBook.AccountCurrency)
             {
-                SecuritySymbol = QuantConnect.Symbol.Empty;
+                SecuritySymbol = null;
```

Comment:
Do we know why `Symbol.Empty` failed serialization to begin with?

Reply:
This bug has been introduced in the last Symbol equality operator change, which now evaluates  `Symbol.Empty == null`. The check at the top of `SymbolJsonConverter.WriteJson()` will now return for both `Symbol.Empty` and `null`.

Reply:
Maybe we can update `SymbolJsonConverter` to use:
`if (ReferenceEquals(symbol, null) { return; }`